test_onedim.TestIonBurnerFlame fails on OSX 10.15.6

<!-- Please fill in the following information to report a problem with Cantera. If you have a question about using Cantera, please post it on our Google Users' Group (https://groups.google.com/forum/#!forum/cantera-users). Feature enhancements should be discussed in the dedicated Cantera enhancements repository (https://github.com/Cantera/enhancements/new/choose) -->

**Problem description**

<!-- A clear and concise description of what the bug is. -->
This issue relates to a second OSX failure reported by @santoshshanbhogue (see #934).

**Steps to reproduce**

<!-- A minimal, complete, and reproducible example demonstrating the problem. See https://stackoverflow.com/help/minimal-reproducible-example for additional suggestions on how to create such an example. -->

Run `scons test`

**Behavior**

<!-- Describe the result of executing the above steps, and how this differs from what you expect to happen. -->

```
======================================================================
FAIL: test_ion_profile (cantera.test.test_onedim.TestIonBurnerFlame)

Traceback (most recent call last):
File "/Users/santosh1/packages/canteraSource/build/python/cantera/test/test_onedim.py", line 1253, in test_ion_profile
self.assertNear(max(self.sim.E), 552.33, 1e-2)
File "/Users/santosh1/packages/canteraSource/build/python/cantera/test/utilities.py", line 57, in assertNear
self.fail(message)
AssertionError: AssertNear: 580.09163013629 - 552.33 = 27.761630136288
Relative error of 4.903055e-02 exceeds rtol = 1.000000e-02
```

**System information**

- Cantera version: [2.5.0b1 commit # 2cd0619] 
- OS: OSX 10.15.6 Catalina
- Python/MATLAB/other software versions: XCode 11.7, python 3.7, numpy 1.17

**Additional context**

Issue was first reported in UG [see thread](https://groups.google.com/g/cantera-users/c/zefZRgeKUbw). Per a comment, @decaluwe had been noticing this for a while.

--- a/interfaces/cython/cantera/test/test_onedim.py
+++ b/interfaces/cython/cantera/test/test_onedim.py
@@ -1251,15 +1251,12 @@ class TestIonBurnerFlame(utilities.CanteraTest):
         self.gas = ct.Solution('ch4_ion.cti')
         self.gas.TPX = Tburner, p, reactants
         self.sim = ct.IonBurnerFlame(self.gas, width=width)
-        self.sim.set_refine_criteria(ratio=4, slope=0.4, curve=0.6)
+        self.sim.set_refine_criteria(ratio=4, slope=0.1, curve=0.1)
         self.sim.burner.mdot = self.gas.density * 0.15
         self.sim.transport_model = 'Ion'
 
-        # stage one
-        self.sim.solve(loglevel=0, auto=True)
-
-        #stage two
-        self.sim.solve(loglevel=0, stage=2, enable_energy=True)
+        self.sim.solve(loglevel=0, stage=2, auto=True)
 
         # Regression test
-        self.assertNear(max(self.sim.E), 552.33, 1e-2)
+        self.assertNear(max(self.sim.E), 591.76, 1e-2)
+        self.assertNear(max(self.sim.X[self.gas.species_index('E')]), 8.024e-9, 1e-2)

